### ISSUE TITLE
Decentralised verification with publically available evidence

### ISSUE THREAD

**Issue by dollhaus** [mastodon/mastodon#25368]
### Pitch

Verification has been discussed before, but only in a seemingly vague sense. I'm proposing a form of decentralised verification that demonstrates the proof to the end user, via a policy of "reputable source".

This was inspired by a status from Jeri Ryan (aka Seven of Nine) https://mastodon.world/@JeriLRyan/110505035263998178, and a subsequent conversation https://mastodonapp.uk/@northernscrub/110514709206388620.

The basic premise is that a user should be able to see that an account is verified, without leaving the feed on which a status is displayed. This improves user experience, and makes it more obvious who is verified and who is not. Currently, this process is limited to third-party tools, which may or may not be reliable.

### How would this be implemented?

This can be accomplished by adding an optional "verified" tag next to the verified account's username, like so: 
![verifiedtagline](https://github.com/mastodon/mastodon/assets/33754407/a24d2a61-cef5-4275-a1b0-64a489d91f7f)

And the verification proof would appear on mouseover, like so:
![verificationproof](https://github.com/mastodon/mastodon/assets/33754407/7d35297a-5ea2-4c7f-9e4a-db4a0a6e96de)

For an organisation, instead of an image, the mouseover event would simply state "Click to see this organisation's social media verificaiton page", and upon clicking, would open that organisation's linked page in a new tab or window. 

### How would this be managed?

For the user seeking verification, a simple form with a toggle to switch between verification of an individual or an organisation would suffice. The former would require an image to be uploaded, the latter a link to that organisation's well-known presence added.

I'd propose that each mastodon instance be granted the authority to verify users. The administrator of said instance, or a nominated individual, would independently vet the request, and approve or deny the request with a single button. It would be prudent to permit moderators to take part in this process as well.

### Motivation

Anyone with a significant public presence has a need for verification on Mastodon. The federated nature of Mastodon means that without this verification, any individual can visit any _other_ Mastodon instance, create an account in the name of the personality they are impersonating, and potentially convince users that they are, in fact, the real deal. The same can be said for practically any organisation that has, or plans on having, a presence on Mastodon. A small organisation could be easily defamed in such a fashion, and in turn seek legal recourse against persons associated with that Mastodon instance. An individual could be greatly damaged by such a turn of events.

In short, verification must be both easily accessible, and easily implementable. The issue has already been raised by an individual with some public presence, and is likely to be raised by others.

**Comment by RodNoc42** [comment]
So I install a mastodon-instance and then I am allowed to verify all my fake-accounts? Don't think that this is a good idea...

In the old times there was a possibility to get free User-Certificates, I don't know the name of the organisation anymore. But all over the world there were trusted community-members. You meet one of them in your city, show your ID and then he knew you are real and signed your cert. I don't know how practicable this is, but I don't think that anyone with low admin-skills can verify everybody is a good idea

**Comment by dollhaus** [comment]
> So I install a mastodon-instance and then I am allowed to verify all my fake-accounts?

No, the publicly-accessible verification evidence is specifically in place to prevent this. You could even add something to flag the verification evidence as false - perhaps setting a threshhold after which an automated message would inform viewers that "a number" of people have flagged that evidence as potentially suspect.

You could even add some checks to activitypub itself, ensuring that any instance which alters how evidence is handled cannot share that evidence with any other instance, thus preventing the suppression of such "context" messages. The whole idea is to make any evidence **publically visible and verifiable** whilst also creating an easily accessible standard that remains entirely the same across any mastodon installation. This ensures that any user, from any instance, knows how to check that the person claiming to be an individual actually is that same individual.

**Comment by 4e554c4c** [comment]
If someone does not have a website that they wish to verify their relation to, then what is there to verify?
What relationship needs verification?
